invenio_oaiharvester/manage.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""CLI tool to harvest records from an OAI-PMH repository.
+
+The output can be directed in several ways:
+   * Direct to files in a directory
+   * Harvested data passed into a "workflow" (See Invenio workflow module)
+   * Printed to stdout (default)
+
+Usage: inveniomanage oaiharvester get <args>
+    -m --metadataprefix <the metadata prefix for the returned records (e.g. 'oai_dc')>
+    -n --name <the name of the OaiHARVEST object that represents the repo (e.g. 'my_source_config')>
+    -s --setSpec <defines the set criteria of the repository (e.g. 'physics:hep-lat' from arXiv.org)>
+    -i --identifiers <identifiers for specific records to harvest (e.g 'oai:arXiv.org:1507.03011')>
+    -f --from <lower bound date for selective harvesting>
+    -t --to <upper bound date for selective harvesting>
+    -u --url <the url for the OAI repo to harvest>
+    -o --output <type of the output: workflow, dir/directory or stdout (default)>
+    -w --workflow <the name of the workflow to be used>
+    -d --dir <the directory where the harvested records should be saved>
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from invenio.ext.script import Manager
+
+from .errors import IdentifiersOrDates
+from .tasks import get_specific_records, list_records_from_dates
+
+manager = Manager(usage=__doc__)
+
+
+@manager.option('-m', '--metadataprefix', dest='metadata_prefix', default=None)
```

Comment:
if you have same options in `get` and `queue` it would be nice to store them in new variable e.g. `harvesting_options`.